openapi-generator/src/main/kotlin/org/jellyfin/openapi/builder/model/EmptyModelBuilder.kt

```diff
@@ -20,6 +21,15 @@ class EmptyModelBuilder(
 	override fun build(data: EmptyApiModel): JellyFile {
 		return TypeSpec.classBuilder(data.name.toPascalCase(from = CaseFormat.CAPITALIZED_CAMEL))
 			.apply {
+				data.interfaces.forEach { interfaceName ->
+					addSuperinterface(
+						ClassName(
+							Packages.MODEL,
+							interfaceName.toPascalCase(from = CaseFormat.CAPITALIZED_CAMEL)
```

Comment:
It seems that capitalized camel case and pascal case are absolutely identical, so I think this can be dropped.
See: https://github.com/pearxteam/kasechange/blob/6c274238ddae339b7cd0d50751855b710facf223/src/commonMain/kotlin/net/pearx/kasechange/StringExtensions.kt#L40